yarn-project/archiver/src/archiver/config.ts

```diff
@@ -55,7 +55,7 @@ export const archiverConfigMappings: ConfigMappingsType<ArchiverConfig> = {
   archiverPollingIntervalMS: {
     env: 'ARCHIVER_POLLING_INTERVAL_MS',
     description: 'The polling interval in ms for retrieving new L2 blocks and encrypted logs.',
-    ...numberConfigHelper(1_000),
+    ...numberConfigHelper(500),
```

Comment:
Just a heads up that this will double the rate at which we query eth nodes/APIs. :moneybag: (times as many validators + boot node + prover node run)

Reply:
Good point. I'll bring it up on standup to discuss.